modules/integration/src/test/scala/scala/cli/integration/NativePackagerTests.scala

```diff
@@ -214,7 +214,7 @@ class NativePackagerTests extends munit.FunSuite {
       // format: off
       val cmd = Seq[os.Shellable](
         TestUtil.cli,
-        "package", helloWorldFileName,
+        "package", "--jvm", "8", helloWorldFileName,
```

Comment:
Maybe we should just change the base image in [`Package.scala`](https://github.com/VirtusLab/scala-cli/blob/a95f8925f636cc8bb4efadeae9c0c7942c85465f/modules/cli/src/main/scala/scala/cli/commands/Package.scala#L355).

Reply:
Done, but this does not fully resolve the problem. I opened an issue #413